services/events/__init__.py

```diff
@@ -98,6 +132,11 @@ def set_default_recurrence_details():
         'num_recurrences': constants.MIN_RECURRENCE
     }
 
+def set_none_recurrence_details():
```

Comment:
Even in the case where there are no recurrences there is still one occurrence of the event.  Part of the logic in the set_occurrences function is that it sets these values accurately